src/system/Form/Radio/RadioOption.tsx

```diff
@@ -0,0 +1,60 @@
+/** @jsxImportSource theme-ui */
+
+import classNames from 'classnames';
+import React from 'react';
+import { Box } from 'theme-ui';
+
+import { VIP_RADIO, type RadioOptionProps } from './Radio';
```

Comment:
Curious if there's an advantage to keeping all the types together vs keeping them with their component